File: Backend/Enities/Issue.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EventSourceDemo {
    public class Issue {

        [Key] public Guid id { get; set; } = Guid.NewGuid();
        [Required] public string title { get; set; } = null!;
        [Required] public string description { get; set; } = null!;
        [Required] public State state { get; set; } = null!;
    }
}

File: lit.frontend/src/Issue.ts
export default class Issue {
    public id: string = "";
    public number: number = 0;
    public title: string = "";
    public description: string = "";

    constructor(id: string, number: number, title: string, description: string) {
        this.id = id;
        this.number = number;
        this.title = title;
        this.description = description;
    }
}
